src/toast/utils.py

```diff
@@ -36,6 +36,56 @@
 )
 
 
+def set_numba_openmp():
+    have_numba_omp = False
+    have_numba_tbb = False
+    try:
+        from numba.npyufunc import tbbpool
+
+        have_numba_tbb = True
+    except ImportError:
+        # no TBB
+        pass
+
+    try:
+        from numba.npyufunc import omppool
+
+        have_numba_omp = True
+    except ImportError:
+        # no OpenMP support
+        pass
+
+    try:
+        from numba import config, njit, threading_layer
+
+        if have_numba_omp:
+            config.THREADING_LAYER = "omp"
+        elif have_numba_tbb:
+            config.THREADING_LAYER = "tbb"
+        else:
+            config.THREADING_LAYER = "default"
+
+        # In order to get numba to actually select a threading layer, we must
+        # trigger compilation of a parallel function.
+        @njit(parallel=True)
+        def foo(a, b):
+            return a + b
+
+        x = np.arange(10.0)
+        y = x.copy()
+        foo(x, y)
+
+        # Log the layer that was selected
+        layer = threading_layer()
+        log = Logger.get()
+        log.info("Numba threading layer set to:  {}".format(layer))
+    except ImportError:
+        # Numba not available at all
+        log = Logger.get()
+        log.info("Numba not available, not setting threading layer")
+    return
```

Comment:
empty returns are not necessary, they look pretty awkward for python programmers. even though I admit they serve the purpose of more easily understand where the function ends beside indentation.

Reply:
I agree empty returns are not necessary, and I have a habit of using them anyway since it makes it easier for my eyes to catch the end of the function.  I'll remove it.